docs/contributing/license.md

```diff
@@ -0,0 +1,44 @@
+# License
+
+Autoware is licensed under Apache License 2.0.
+Thus all contributions will be licensed as such as per clause 5 of the Apache License 2.0:
+
+```text
+5. Submission of Contributions. Unless You explicitly state otherwise,
+   any Contribution intentionally submitted for inclusion in the Work
+   by You to the Licensor shall be under the terms and conditions of
+   this License, without any additional terms or conditions.
+   Notwithstanding the above, nothing herein shall supersede or modify
+   the terms of any separate license agreement you may have executed
+   with Licensor regarding such Contributions.
+```
+
+Here is an example copyright header to add to the top of a new file:
+
+```text
+Copyright [first year of contribution] The Autoware Contributors
```

Comment:
Should I remove the year here?
https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/

```suggestion
Copyright The Autoware Contributors
```

Reply:
I think it is redundant. We can remove it. It's history will be tracked through git anyway.

Reply:
Actually, I couldn't find enough examples of doing this. Some files used this style, but they are archived.
https://github.com/LF-Engineering/easycla-api/blob/fc32405b59852ef69ee9c003950d7663eebcb0b9/cmd/utils.go

Also, seeing https://www.apache.org/legal/src-headers.html#faq-whyheader, I think it's good to include all necessary information into files.

> License headers allow someone examining the file to know the terms governing use of the work, even when it is distributed without the rest of the distribution. Without a licensing notice, it must be assumed that the author has reserved all rights, including the right to copy, modify, and redistribute.

@xmfcx So, can I revert this change?

Reply:
Since we won't maintain the latest year, I believe it's not so troublesome.

Reply:
Fine, but I still think it is unnecessary information.

Reply:
I know, me too. But I'd like to avoid unnecessary troubles in the future, so just in case!

Reply:
@xmfcx It's easy to erase old years, but the reverse isn't easy.
Let's remove them after we can be sure they are completely unnecessary.

Reply:
Could you approve this again? :pray: